tests/manage/z_cluster/nodes/test_disk_failures.py

```diff
@@ -377,6 +398,10 @@ def test_recovery_from_volume_deletion(self, nodes, pvc_factory, pod_factory):
             f"{[osd_pod.ocp.get_resource(pod.get().get('metadata').get('name'), 'STATUS') for pod in get_osd_pods()]}"
         )
 
+        # We need to silence the old osd crash warning due to BZ https://bugzilla.redhat.com/show_bug.cgi?id=1896810
```

Comment:
this is a workaround. Please follow https://ocs-ci.readthedocs.io/en/latest/workarounds.html